scss_test.scss

```diff
@@ -0,0 +1,4 @@
+.parent {
+  display: inline-block;
+  background: #fff;
```

Comment:
Expected `display`, found `background`                                      property-sort-order<br>Color literals such as '#fff' should only be used in variable declarations  no-color-literals